ives_libby_ndvi.py

```diff
@@ -0,0 +1,1615 @@
+    "\n",
+    "* NDVI on the x axis and formatted dates on the y for both NEON sites on one figure/axis object.\n",
+    "* Each site should be identified with a different color in the plot and legend.\n",
+    "* The final plot **data should be cleaned to remove the influence of clouds**.\n",
+    "* Be sure to include appropriate title and axes labels.\n",
+    "\n",
+    "Add additional cells as needed for processing data (e.g. defining functions, etc), but be sure to:\n",
+    "* follow the instructions in the code cells that have been provided to ensure that you are able to use the sanity check tests that are provided. \n",
+    "* include only the plot code in the cell identified for the final plot code below"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": false,
+    "editable": false,
+    "nbgrader": {
+     "cell_type": "markdown",
+     "checksum": "ce17d4d685cd4c7034bd7b0bb389342a",
+     "grade": false,
+     "grade_id": "single-scene-instructions",
+     "locked": true,
+     "schema_version": 3,
+     "solution": false,
+     "task": false
+    }
+   },
+   "source": [
+    "## Task 1: \n",
+    "\n",
+    "In the cell below, create a single dataframe containing MEAN NDVI, the site name, \n",
+    "and the date of the data for the HARV site \n",
+    "scene `HARV/landsat-crop/LC080130302017031701T1-SC20181023151837`.  The column names for the  final\n",
+    "DataFrame should be`mean_ndvi`, and `site`, and the data should be **indexed on the date**. \n",
+    "\n",
+    "Use the functions that we reviewed in class (or create your own versions of them) to implement your code\n",
+    "\n",
+    "### In the Cell below Place  All Functions Needed to Run this Notebook (20 points)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "deletable": false,
+    "editable": false,
+    "nbgrader": {
+     "cell_type": "code",
+     "checksum": "0ee340cd9af6c949a08eb4b325716ae0",
+     "grade": false,
+     "grade_id": "cell-618e3588853f3ed8",
+     "locked": true,
+     "schema_version": 3,
+     "solution": false,
+     "task": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "### DO NOT REMOVE THIS LINE OR EDIT / MOVE THIS CELL ###\n",
+    "start_time = datetime.now()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "deletable": false,
+    "nbgrader": {
+     "cell_type": "code",
+     "checksum": "653ebd5db668245408615979f6c20944",
+     "grade": true,
+     "grade_id": "function-definitions-check",
+     "locked": false,
+     "points": 40,
+     "schema_version": 3,
+     "solution": true,
+     "task": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def calculate_mean_ndvi_clouds(folder_path):\n",
+    "    '''\n",
+    "\n",
+    "    Parameters\n",
+    "    ----------\n",
```

Comment:
Missing a brief function description string here , before parameters, summarizing each of the steps commented out below. (As you did with your other functions!)